### PR TITLE
[MIRROR] legion cores remove CC instead of adminhealing [READY]

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/42378
----------------------------

[unimaginative icon shown](https://imgur.com/a/pwWxOsx)
Legion cores are far stronger than any comparable healing item. Patches take time to self-apply, yet heal less than a legion core, and don't have many of the side effects (such as removing diseases, purging chems, resetting hunger). 

Further, large numbers of legion cores trivialise boss fights. Spamming legion cores allows you to tank a boss's damage for a very long time - carrying a backpack full of ten effectively gives you eleven health bars. On the other hand, without an item to quickly heal and remove damage slowdown many megafauna fights would become prohibitively difficult. 

I believe this is a good compromise - cores still heal a moderate amount, meaning that legion tendrils are not impossible, and remove damage slowdown meaning that megafauna fights don't require flawless play. The damage slowdown immunity is given as a status effect to reduce the difference between a person who has stocked up on hundreds of cores and someone who has a few - the person with hundreds of cores will have more healing, but similar amounts of slowdown reduction. You can also implant the core for one get out of jail free card.

Lorewise for those who care this will be fluffed as legion cores "holding you together", but not fixing the underlying issues.

todo: 

- [x] testing
- [x] status effect icon
- [x] flavour text/lore

### CODE REVIEW

**Comment by GuyonBroadway** [comment]
>10 fucking files changed to implement your unwanted nerf.


That's how tg works.

**Comment by Jujumatic** [comment]
what the fuck is this pr this is why i hate tg smh

File: code/modules/mining/equipment/regenerative_core.dm [hunk 1]
```diff
@@ -78,12 +78,12 @@
 				to_chat(user, "<span class='notice'>[src] are useless on the dead.</span>")
 				return
 			if(H != user)
-				H.visible_message("[user] forces [H] to apply [src]... [H.p_they()] quickly regenerate all injuries!")
+				H.visible_message("[user] forces [H] to apply [src]... Black tendrils entangle and reinforce [H.p_them()]!")
 				SSblackbox.record_feedback("nested tally", "hivelord_core", 1, list("[type]", "used", "other"))
 			else
-				to_chat(user, "<span class='notice'>You start to smear [src] on yourself. It feels and smells disgusting, but you feel amazingly refreshed in mere moments.</span>")
+				to_chat(user, "<span class='notice'>You start to smear [src] on yourself. Disgusting tendrils hold you together and allow you to keep moving, but for how long?</span>")
```
Comment (Jujumatic) [inline code/modules/mining/equipment/regenerative_core.dm]: the person who made this pr confirmed tentacle fetish

**Comment by DaydreamPuppeh** [comment]
-1.

**Comment by Carbonhell** [comment]
-1

**Comment by GuyonBroadway** [comment]
Reverting this shit inline cus all it takes is three lines.

I could even do it with just ONE line if it didn't include the to_chat stuff.

**Review by Carbonhell — APPROVED** [review]

**Review by Carbonhell — APPROVED** [review]

**Pull request merged**